doc/Settings.md

```diff
@@ -106,6 +106,18 @@ See [details on telemetry](../README.md#datatelemetry), and our [primary privacy
 
 If set to true, the `telemetry.disable` setting will prevent any event from being written by the program.
 
+## Logging
+
+The `logging` settings control the level of detail in log files. `--verbose-logs` will override this setting and always creates a verbose log.
```

Comment:
nit: maybe also mention the default setting is info. Or add "default" in json schema.

Reply:
Added